### PR TITLE
Update en.yml file with marketplace listing content

### PR DESCRIPTION
**Pull request by pmgrove**
:v:

/cc @zendesk/vegemite, @zendesk/i18n 

### Description

Update en.yml file with marketplace listing content

### References
* JIRA: n/a

### Risks
* [low] - app content may not show on marketplace

### CODE REVIEW

File: translations/en.yml [hunk 1]
```diff
@@ -5,6 +5,40 @@ packages:
 
 parts:
   - translation:
+      key: "txt.apps.highrise_app.app.name"
+      title: "Name for App: Highrise"
+      value: "Highrise"
```
Comment (injuhwang) [inline translations/en.yml]: Please add `hidden: true`.

File: translations/en.yml [hunk 2]
```diff
@@ -5,6 +5,40 @@ packages:
...
+  - translation:
+      key: "txt.apps.highrise_app.app.short_description"
+      title: "Description for App: Highrise"
```
Comment (injuhwang) [inline translations/en.yml]: Please add the following:
> Do not translate \"Highrise.\"

File: translations/en.yml [hunk 3]
```diff
@@ -5,6 +5,40 @@ packages:
...
+      value: "Highrise profiles in tickets, with sync and search"
+  - translation:
+      key: "txt.apps.highrise_app.app.long_description"
+      title: "Long Description for App: Highrise"
```
Comment (injuhwang) [inline translations/en.yml]: Please add the following:
> Do not translate \"Highrise.\"

**Comment by pmgrove** [comment]
@injuhwang 💇

**Comment by injuhwang** [comment]
👍 for the strings

**Comment by sandlerr** [comment]
:shipit:! Cos I'm still getting github email for all these prs that were
opened in June :D

**Comment by ocke** [comment]
You can checkout anytime you want, but you can never leave!

**Pull request merged**